static/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -165,13 +165,36 @@ class ConfigureIntegration extends DeprecatedAsyncView<Props, State> {
       addErrorMessage(t('Something went wrong! Please try again.'));
     }
   };
+
+  handleOpsgenieMigration = () => {
+    this.setState(
+      {
+        plugins: (this.state.plugins || []).filter(({id}) => id === 'opsgenie'),
+      },
+      () => addSuccessMessage(t("This doesn't do anything yet!"))
+    );
+  };
+
+  isInstalledOpsgeniePlugin = (plugin: PluginWithProjectList) => {
+    return (
+      plugin.id === 'opsgenie' &&
+      plugin.projectList.length >= 1 &&
+      plugin.projectList.find(({enabled}) => enabled === true)
```

Comment:
optional nit: Using `.find()` will return the item (which is evaluated to `true`) but if can also return a boolean if you use `.some()` instead and drop the `=== true` which is identical (in this scenario)